app/views/shared/_status.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="<%= dialog_column_class %>">
+  <h1>Status of your request</h1>
+  <hr />
+  <dl class='dl-horizontal dl-invert'>
+    <dt>Requested on</dt>
+    <dd><%= request.created_at.to_formatted_s(:long_ordinal) %></dd>
+  </dl>
+  <hr />
+  <%= render partial: 'shared/searchworks_item_information', locals: { request: request } %>
```

Comment:
Hmm, I feel like I've ran into nothing but trouble using CanCan and inheritance, but it may be worth a try.

I was also thinking of a `current_request` pattern that could be set in each controller.